packages/slice-machine/src/components/SideNav/SideNav.tsx

```diff
@@ -132,6 +132,38 @@ export const SideNavLink: FC<SideNavLinkProps> = ({
   );
 };
 
+export type SideNavButtonProps = {
+  title: string;
+  active?: boolean;
+  Icon: FC<SVGProps<SVGSVGElement>>;
+  RightElement?: ReactNode;
+  component?: "button" | FC<{ onClick?: (event: MouseEvent) => void }>;
+  onClick?: (event: MouseEvent) => void;
+};
+
+export const SideNavButton: FC<SideNavButtonProps> = ({
+  title,
+  RightElement,
+  Icon,
+  active,
+  component: Comp = "button",
+  ...otherProps
+}) => {
+  const visible = useMediaQuery({ max: "medium" });
+
+  return (
+    <Tooltip content={title} side="right" visible={visible}>
+      <Comp {...otherProps} className={styles.link} data-active={active}>
```

Comment:
⚠️ Let's have a proper style management, then. 

`.button` instead of link, and same for other, it will prevent the `button.link` in style + will prevent having link used for button by doing.

```
.link,
.button
```

Where needed

Reply:
Done ✅

Reply:
I just that for that:

```
button.link {
  width: 100%;
}
```

You can do that now:

```
.button
```

Reply:
Missed that one, done ✅